orchagent/orchdaemon.cpp

```diff
@@ -188,7 +188,7 @@ bool OrchDaemon::init()
     vector<table_name_with_pri_t> ports_tables = {
         { APP_PORT_TABLE_NAME,        portsorch_base_pri + 5 },
         { APP_SEND_TO_INGRESS_PORT_TABLE_NAME,        portsorch_base_pri + 5 },
-        { APP_VLAN_TABLE_NAME,        portsorch_base_pri + 2 },
+        { APP_VLAN_TABLE_NAME,        portsorch_base_pri     },
```

Comment:
This is needed for create operation.

Reply:
This won't impact create operation, because VLAN and VLAN member have same priority, and in create operation VLAN create will happen before VLAN_MEMBER create.